### PR TITLE
chore(holidays): australia & malaysia

### PR DESCRIPTION
**Pull request by curbengh**
replace 2021 dates with 2023's

### CODE REVIEW

File: app/src/main/assets/malaysia.ics [hunk 1]
```diff
@@ -320,45 +321,60 @@ RRULE:FREQ=YEARLY;INTERVAL=1
...
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210719
-DTEND;VALUE=DATE:20210720
+DTSTART;VALUE=DATE:20230627
+DTEND;VALUE=DATE:20230628
 UID:mal_59
 STATUS:CONFIRMED
 SUMMARY:Arafat Day (Terengganu)
 END:VEVENT
 END:VCALENDAR
```
Comment (tibbi) [inline app/src/main/assets/malaysia.ics:365]: the file has to end with this END:VCALENDAR and not END:VEVENT

**Comment by tibbi** [comment]
have you tested if this works at all?

File: app/src/main/assets/malaysia.ics (hunk 1 above)
Comment (curbengh) [reply app/src/main/assets/malaysia.ics]: fixed

**Comment by curbengh** [comment]
tested on v6.20.1.228

**Pull request merged**